common/org.testeditor.dsl.common.ui/src/org/testeditor/dsl/common/ui/utils/ProjectContentGenerator.xtend

```diff
@@ -145,6 +145,10 @@ class ProjectContentGenerator {
 	protected def void setupMavenProject(IProject project, String[] fixtures, IProgressMonitor monitor) {
 		var IFile buildFile = project.getFile("pom.xml")
 		buildFile.create(new StringInputStream(getPomContent(fixtures, project.name)), IResource.NONE, monitor)
+		val mavenSettings = System.getProperty("TE.MAVENSETTINGSPATH", "")
+		if (mavenSettings.length > 0) {
```

Comment:
In Xtend you can use `mavenSettings.isNullOrEmpty`. Then there is no need to return an empty default String above.